### ISSUE TITLE
API-26: MicroProfile Context Propagation 1.0 in OpenLiberty

### ISSUE THREAD

**Issue by njr-11** [OpenLiberty/open-liberty#5301]
Implement the MicroProfile Context Propagation 1.0 specification in OpenLiberty.

Feature in Aha: https://bigblue.aha.io/features/API-26

Documentation: https://github.com/OpenLiberty/docs/issues/471
GA Release Blog: #8378
OpenLiberty.io Draft blog post update for GA: https://github.com/OpenLiberty/blogs/pull/65
OpenLiberty.io GA blog post update for GA: https://github.com/OpenLiberty/blogs/pull/68
OpenLiberty.io Blog post updates for original concurrency article: https://github.com/OpenLiberty/docs/issues/473

**Comment by marikaj123** [comment]
According to Nathan, all beta work is complete and went into a green release build

**Comment by marikaj123** [comment]
@aguibert - Please note, when the feature is ready to go into the next GA release please add the label 'target:ga'.

**Comment by skasund** [comment]
L2 has requested STE slides for this feature.  The STE template can be found at the link below: 

https://ibm.box.com/s/1an42g7zdgmaj84w7dft0indqfgi8ffm

Please upload the completed slides to the same STE Archive BOX folder.  Thanks!

**Comment by gscottj** [comment]
This feature does not add any new or change any existing user interfaces. Therefore, no accessibility verification testing is required.

**Comment by skasund** [comment]
@njr-11  Thanks for the slides.  I've provided the STE approval.

**Comment by donbourne** [comment]
**Serviceability Approval Comment** - Please answer the following questions for serviceability approval:

1.  WAD -- does the WAD identify the most likely problems customers will see and identify how the feature will enable them to diagnose and solve those problems without resorting to raising a PMR?  Have these issues been addressed in the implementation?

2.  Test and Demo -- As part of the serviceability process we're asking feature teams to test and analyze common problem paths for serviceability and demo those problem paths to someone not involved in the development of the feature (eg. L2, test team, or another development team).  
a) What problem paths were tested and demonstrated? 
b) Who did you demo to?
c) Do the people you demo'd to agree that the serviceability of the demonstrated problem scenarios is sufficient to avoid PMRs for any problems customers are likely to encounter, or that L2 should be able to quickly address those problems without need to engage L3?

3.  SVT -- SVT team is often the first team to try new features and often encounters problems setting up and using them.  Note that we're not expecting SVT to do full serviceability testing -- just to sign-off on the serviceability of the problem paths they encountered.
a) Who conducted SVT tests for this feature?
b) Do they agree that the serviceability of the problems they encountered is sufficient to avoid PMRs, or that L2 should be able to quickly address those problems without need to engage L3?  

4.  Which L2 / L3 queues will handle PMRs for this feature?  Ensure they are present in the contact reference file and in the queue contact summary, and that the respective L2/L3 teams know they are supporting it.  Ask Don Bourne if you need links or more info.

**Comment by njr-11** [comment]
@donbourne the above serviceability questions are already answered under the corresponding checklist issue for this feature where you had also posted the questions.  See:
https://github.com/openliberty/open-liberty/issues/6676#issuecomment-509771174

**Comment by marikaj123** [comment]
@njr-11 and @aguibert - Does the feature contain any translated files. If yes, are they in the GM build?

**Comment by njr-11** [comment]
@marikaj123 yes, this feature includes translations here,
/com.ibm.ws.concurrent.mp.1.0/resources/com/ibm/ws/concurrent/mp/resources
These messages have all been delivered quite a while ago and the translatable files are present for all of the languages, each including entries for all of the messages.

**Comment by marikaj123** [comment]
@jhanders34 - Please help to approve this feature. 
@chirp1 @Charlotte-Holt, Please help to approve this feature.  
@njr-11 - Nathan was the feature code enabled in Liberty release 19007?

**Comment by njr-11** [comment]
@marikaj123 this feature was beta in 19.0.0.7.  We did not promote it to GA because it still needs approvals.

**Comment by marikaj123** [comment]
keeping target:ga as it may become a candidate for Liberty release 19008.

**Comment by njr-11** [comment]
> keeping target:ga as it may become a candidate for Liberty release 19008.

@marikaj123 I had specifically added target:ga in order to make it a candidate for 19.0.0.8.   It seems like the current process has some ambiguity with the target:ga label when added in advance of the previous release's GA because this is the second time it has caused confusion.  I also noticed that you have added a label for beta:19700, although we first beta'd this back in 19.0.0.2 and had a single beta:19200 label for that, rather than labeling beta on every release since (see the issue labels).

**Comment by marikaj123** [comment]
The note above is mostly for me.  I do not see any confusion with the target:ga. I labelled your feature with beta:19700 so it can go into the Beta 19007 blog query in case any blog work is needed.

**Comment by marikaj123** [comment]
@njr-11 - Thank your for the delivery of the feature, tagging with release:19008.
